receiver/httpreceiver/README.md

```diff
@@ -0,0 +1,60 @@
+# HTTP Receiver
+This receiver is capable of collecting logs for a variety of services, serving as a default HTTP log receiver. Anything that is able to send JSON structured logs to an endpoint using HTTP will be able to utilize this receiver.
+
+## Supported Pipelines
```

Comment:
Can we add a minimum collector version section? See other components for how this section should look.